test/examples/17-resizable-handles.jsx

```diff
@@ -42,7 +42,7 @@ export default class ResizableHandles extends React.PureComponent {
         w: 2,
         h: y,
         i: i.toString(),
-        resizeHandles: _.shuffle(availableHandles).slice(0, _.random(1, availableHandles.length-1))
+        resizeHandles: availableHandles
```

Comment:
This was intentional. I'm using this in the tests and wanted to rely on handles being there on all cardinal directions.